docs/content/how-to/disconnected/known-issues.md

```diff
@@ -44,3 +45,148 @@ data:
 ```
 
 - **The HO was deployed using the Hypershift binary:** In this case you will just need to remove the flag `--enable-uwm-telemetry-remote-write` from the hypershift deployment command.
+
+## HCP CLI failing to create an hosted cluster with `failed to extract release metadata: failed to get repo setup: failed to create repository client for <your.internal.registry>
+
+See: [CNV-38194](https://issues.redhat.com/browse/CNV-38194)
+Currently the HCP client is directly (client side) trying to access the OCP release image, this implies some additional requirements on client side:
+
+1. the HCP CLI client should be able to directly reach the internal mirror registry, and this could require additional configuration (a proxied access to the api server of the management server is not enough)
+1. the HCP CLI client should explicitly consume the pull secret of the internal mirror registry
+1. the HCP CLI client should explicitly consume the internal CA used to sign the TLS cert of the internal mirror registry
+
+**Workaround:**
+Ensure that you execute `hcp create cluster` command on a bastion host properly configured to access the internal registry.
+You can eventually add `--additional-trust-bundle <filename>` parameter with the CA used to sign the cert of your internal registry if needed.
```

Comment:
`you can eventually add ` is this waiting on a ticket to add this? If so, could you please link the ticket here?

Reply:
This will also be implicitly handled as part of https://issues.redhat.com/browse/CNV-38194 which is mentioned above.